.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -14,4 +14,4 @@ If tests do fail, click on the red `X` to learn why by reading the logs.
 Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software
 
 Thanks!
-@publiclab/is-reviewers
+@publiclab/is-reviewers please review my pr.
```

Comment:
Uh, can you calitalize PR to make it look final and professional (i guess). Also can you add some emoji to make the review request look friendly somehow?